src/components/shared/ListOf/index.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { useCharityAPI } from '../../../clients';
+import './style.scss';
+import { useSpring, animated } from 'react-spring';
+import useMedia from '../../../Helpers/useMedia';
+import { Loader, ButtonLoader, ListItemLoader } from './myLoader';
+import { Widget } from '../Widget';
+
+const ListItem = ({ data, timeElapsed }) => {
+  const [isHover, setHover] = useState();
+  const hovered = useSpring({
+    transform: isHover ? 'translateX(15px)' : ' translateX(0px)'
+  });
+  const {
+    link: { url: linkUrl },
+    id,
+    createdAt,
+    image: { url: imgUrl, alternativeText },
+    description
+  } = data;
+  return (
+    <animated.a
+      href={linkUrl}
+      key={id}
+      className="supporter flex flex-col"
+      style={hovered}
+      onMouseEnter={() => {
+        setHover(true);
+      }}
+      onMouseLeave={() => {
+        setHover(false);
+      }}
+    >
+      <article className="flex">
+        <div className="supporter__image-container">
+          <img
+            className="supporter__image w-full h-full"
+            src={imgUrl}
+            alt={alternativeText}
+          />
+        </div>
+        <div className="pl-4 flex flex-col">
+          <p className="text-c200 text-sm leading-loose mb-1">
+            {timeElapsed(createdAt)}
+          </p>
+          <p className="font-bold text-c100 leading-relaxed">{description}</p>
+        </div>
+      </article>
+    </animated.a>
+  );
+};
+
+const ListOf = ({ data, loading, error }) => {
```

Comment:
Too generic name for very specific component!

Reply:
I know :/ .... I spent a lot of time trying to find a suitable name and couldn't find a proper one